api/src/routes/auth.js

```diff
@@ -35,4 +37,18 @@ router.post('/login', (req, res) => {
   });
 });
 
+router.post('/q-and-a', (req, res) => {
+  Questions.create((err) => {
+    if (err) {
+      return res.sendStatus(500)
+    }
+  });
+
+  Response.create((err) => {
+    if (err) {
+      return res.sendStatus(500)
+    }
+  });
+});
```

Comment:
How does this know what is a Question and what is a response if its under the same endpoint? Wouldn't this create a questions and a response since this is both sequential?

Reply:
> How does this know what is a Question and what is a response if its under the same endpoint? Wouldn't this create a questions and a response since this is both sequential?

as for this the redesign it should look something like this 
![image](https://github.com/user-attachments/assets/28d1f2ce-c22a-4d34-96bc-d1c60cfee357)
 (screenshot from my cs422 project, this uses posgreSQL)